CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 * Upgrade to pulumi-terraform-bridge v2.3.3
+* Update to v2.10.0 of the AzureRM Terraform Provider
```

Comment:
BTW - as a general thing - and I expect this may add some work required - it would be very nice to hyperlink to changelog entry of the upstream provider here.